libs/auth-api-lib/src/lib/entities/dto/address.dto.ts

```diff
@@ -0,0 +1,21 @@
+import { ApiPropertyOptional } from '@nestjs/swagger'
+
+export class AddressDTO {
+  @ApiPropertyOptional({ type: String })
```

Comment:
Feel like the `type` property is redundant.
But we need to set the `nullable` property to `true`. As the `@ApiPropertyOptional()`  only sets `required: false`  and default nullable is not allowed.

Reply:
I'm pretty sure `type` is neccessary when typescript is `strict`, since then it can't reflect the "string" part of the `string | null` union.

In any case, I decided to get rid of all nulls from this API since we don't want null claims anywhere.